build/docker/intel-idxd-initcontainer.Dockerfile

```diff
@@ -12,38 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:unstable AS builder
+FROM debian:unstable-slim AS builder
 
-RUN apt-get update && apt-get install -y \
-            build-essential autoconf automake autotools-dev libtool \
-            pkgconf asciidoc xmlto uuid-dev libjson-c-dev libkmod-dev \
-            libudev-dev libkeyutils-dev curl
+RUN echo "deb-src http://deb.debian.org/debian unstable main" >> \
+        /etc/apt/sources.list.d/deb-src.list && \
+    apt update && apt install -y --no-install-recommends \
+        gcc make patch autoconf automake libtool pkg-config \
+        libjson-c-dev uuid-dev curl ca-certificates && \
+    cd /usr/local/src/ && apt --download-only source uuid libjson-c5 && cd /
 
 ARG ACCEL_CONFIG_VERSION="3.4.2"
 ARG ACCEL_CONFIG_DOWNLOAD_URL="https://github.com/intel/idxd-config/archive/accel-config-v$ACCEL_CONFIG_VERSION.tar.gz"
 ARG ACCEL_CONFIG_SHA256="9cb2151e86f83949a28f06a885be3bf3100906f9e3af667fa01b56e7666a3c1c"
 
-RUN curl -fsSL "$ACCEL_CONFIG_DOWNLOAD_URL" -o accel-config.tar.gz \
-    && echo "$ACCEL_CONFIG_SHA256 accel-config.tar.gz" | sha256sum -c - \
-    && tar -xzf accel-config.tar.gz \
-    && rm accel-config.tar.gz
+RUN curl -fsSL "$ACCEL_CONFIG_DOWNLOAD_URL" -o accel-config.tar.gz && \
+    echo "$ACCEL_CONFIG_SHA256 accel-config.tar.gz" | sha256sum -c - && \
+    tar -xzf accel-config.tar.gz
+
+ADD demo/accel-config-kmod.patch /
```

Comment:
time to add `demo/idxd-config` (or `accel-config`) sub-directory for all intel-idxd-initcontainer related content?